### PR TITLE
Add a shared php coding style fixer config

### PR DESCRIPTION
**Pull request by ChristophWurst**
As requested by @nickvergessen.

This is a minimalist version of a config. We can slowly get more strict with more rules.

You can run this via

```bash
composer i
composer run cs:fix
```

If that works well we can also document it in our manual for apps.

### CODE REVIEW

File: build/files-checker.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
 	'.idea',
 	'.jshintrc',
 	'.mailmap',
+	'.php_cs.dist',
```
Comment (ChristophWurst) [inline build/files-checker.php]: release team notified about the change

File: composer.json [hunk 2]
```diff
@@ -15,9 +15,12 @@
     },
     "require-dev": {
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "christophwurst/nextcloud-coding-standard": "^0.0.1"
```
Comment (nickvergessen) [inline composer.json]: move to nextcloud, like the eslint configs?

**Review by kesselb — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (ChristophWurst) [reply composer.json]: There is no nextcloud org for packagist. I only have a personal one

Comment (ChristophWurst) [reply composer.json]: I can try to acquire nextcloud/ https://packagist.org/about#naming-your-package

Comment (ChristophWurst) [reply composer.json]: Bildschirmfoto von 2020-03-31 11-38-27

Hey @PhrozenByte could you please add me as maintainer to your package so we can package ours? :v:

Comment (PhrozenByte) [reply composer.json]: Don't even remember creating that one :confused:

Done :+1:

**Review by rullzer — APPROVED** [review]

**Pull request merged**